src/modules/drawing/lib/style.js

```diff
@@ -49,6 +53,12 @@ const sketchPoint = new Circle({
     fill: whiteSketchFill,
 })
 
+export const gpxStyle = new Style({
+    fill: redFill,
+    stroke: redStroke,
+    image: point,
+})
+
```

Comment:
And for this one a gpx style file `src/utils/gpxStyleUtils.js`

Or put all style utils in `src/utils/styleUtils.js`